checkstyle.xml

```diff
@@ -154,6 +154,11 @@
         <module name="MultipleVariableDeclarations"/>
         <module name="SimplifyBooleanExpression"/>
         <module name="SimplifyBooleanReturn"/>
+        <module name="FinalLocalVariable">
+            <property name="tokens" value="VARIABLE_DEF,PARAMETER_DEF"/>
```

Comment:
Remove this line, it's pointless, as we require all parameters to be final with that other rule anyway.

Reply:
Hmm… it seems to make a significant difference in amount of warnings